base_vat_sanitized/models/res_partner.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+
+from odoo import models, fields, api
+import re
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    sanitized_vat = fields.Char(
```

Comment:
Some users may want to keep the "spacing" of the VAT number. The approach is exactly the same as the field "sanitized_acc_number" of the "base" module.